src/parsed_codeblock.py

```diff
@@ -0,0 +1,31 @@
+from enum import Enum
+from pathlib import Path
+
+class CodeblockSeparator(Enum):
+    BACKTICKS = 1
```

Comment:
didn't use that, delete 

create issue for multiple `CodeblockSeparator` support -> #6